### PR TITLE
Fix the color picking issue in version 1.1.3 and angular 10 migration.

### PR DESCRIPTION
**Pull request by lucboutier**
See my comments on issue 263 for more details.

Let me know if there is any problem with this PR.

### CODE REVIEW

**Comment by alexgurrola** [comment]
@kolkov This appears to be related to: https://github.com/kolkov/angular-editor/issues/278, https://github.com/kolkov/angular-editor/issues/273, https://github.com/kolkov/angular-editor/issues/263, and https://github.com/kolkov/angular-editor/issues/258.

**Comment by argelj289** [comment]
the color I selected when using color picker is not applied to text or its background in angular version 10

**Comment by lucboutier** [comment]
@argelj289 If you used this patch you should tell more on how to reproduce because I use this in my project that is using angular 10 and we don't have any issues.

**Comment by kolkov** [comment]
> See my comments on issue 263 for more details.
> 
> Let me know if there is any problem with this PR.

Thanks for the PR, but I think that we need one service instance for each component if multiple editors placed on one page.

File: projects/angular-editor/src/lib/angular-editor.component.ts [hunk 1]
```diff
@@ -34,8 +34,7 @@ import {isDefined} from './utils';
       provide: NG_VALUE_ACCESSOR,
       useExisting: forwardRef(() => AngularEditorComponent),
       multi: true
-    },
-    AngularEditorService
+    }
```
Comment (kolkov) [inline projects/angular-editor/src/lib/angular-editor.component.ts:37]: I think we not need to remove this line.  Only in ToolbarComponent.

Comment (lucboutier) [reply projects/angular-editor/src/lib/angular-editor.component.ts:37]: Well as AngularEditorService defines:

`@Injectable({
  providedIn: 'root'
})`

It is not useful to keep it in the component. Or maybe we should remove the providedIn: 'root' which could be cleaner for a lib.

Comment (lucboutier) [reply projects/angular-editor/src/lib/angular-editor.component.ts:37]: Actually especially if a single instance of the service is required per component it should not be defined neither in the module neither in root actually and you should have the provided back...

**Comment by lucboutier** [comment]
Sorry, there have been a commit on our master branch that we did not wanted to be included in this PR. Should have branched..
We actually forked the component for now as we upgraded also font-awesome and switched the styles to use bootstrap (I don't think you want that on your component so that was not intended to be part of this PR).

I will close this P.R. and make a new one later (hopefully with a fix for the AngularEditorService usages that are mixed-up.

**Comment by kolkov** [comment]
> Извините, в нашей основной ветке есть фиксация, которую мы не хотели включать в этот PR. Должен был быть разветвлен ... На данный момент
> мы фактически разделили компонент, поскольку мы обновили также font-awesome и переключили стили на использование начальной загрузки (я не думаю, что вы хотите, чтобы это было в вашем компоненте, поэтому это не должно было быть частью этого PR) .
> 
> Я закрою этот PR и сделаю новый позже (надеюсь, с исправлением неправильного использования AngularEditorService.

Ok. Thanks fo your contribution!

**Pull request closed without merging**